docs/docs/native-docker.md

```diff
@@ -1,17 +1,64 @@
 ---
-title: Running Containers on Marathon
+title: Provisioning Containers
 ---
 
-Marathon supports Docker and Appc container images by either one of two runtimes; Docker Engine and Universal Container Runtime.
+# Provisioning Containers
 
-# Running Docker Containers on Marathon
+A containerizer is a Mesos agent component responsible for launching containers, within which you can run a Marathon app. Running apps in containers offers a number of benefits, including the ability to isolate tasks from one another and control task resources programmatically.
 
-Marathon enables users to run Docker container images with two different runtimes:
+Marathon enables users to run container images with two different runtimes:
 
-  1. [Docker containerizer](# Docker Containerizer) using the native Docker Engine as runtime.
-  2. Mesos containerizer using the Universal Container Runtime.
+1. Mesos containerizer using the [Universal Container Runtime](#ucr).
+1. [Docker containerizer](#docker-containerizer) using the native Docker Engine as runtime.
```

Comment:
markdown will render this list correctly -- if you start lists all with 1, then if you have to add something, you don't have to redo all the numbers.